index.php

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+
+
+    <title>Login - Blog</title>
+  </head>
+  <body>
+    <h1>Login - Blog</h1> <br>
+    <form>
+  <div class="form-group">
+    <label for="exampleInputEmail1"> <b>E-mail</label></b>
```

Comment:
O correto é ...

`<label for="exampleInputEmail1"> <b>E-mail</b></label>`

Aplicar a mesa estrutura HTML nas outras labels.